cpp/translator.cc

```diff
@@ -243,11 +243,40 @@ bool IsEmptyEntryWithoutTranslator(const std::string &entry) {
 }
 
 
+void SetupVuFindSortLimit(DbConnection &db_connection, std::string * const create_sort_limit, const std::string &offset,
+                          const bool use_untranslated_filter)
+{
+    if (use_untranslated_filter) {
+        const std::string translator=GetTranslatorOrEmptyString();
+        if (translator.empty()) {
+            ShowErrorPage("Error - No Valid User", "Not valid user selected");
+            std::exit(0);
+        }
+        const std::string translated_by_translator("CREATE TEMPORARY TABLE translated_by_translator (INDEX (token)) AS "
+                                                   "(SELECT DISTINCT token FROM vufind_ger_sorted WHERE translator=\'" + translator + "\')");
```

Comment:
Why do you escape single quotes in a C-style string?  Please don't or use double quotes.